File: Data Structures & Algorithms/majority-element/submission-1.ts
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    majorityElement(nums: number[]): number {
      const max = Math.floor(nums.length/2)
      for(let num of nums){
        let count = nums.reduce((acc, val) => acc + (val === num ? 1 : 0), 0)
        if(count > max){
          return num
        }
      }
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int n = nums.length, res = 0;
        int cnt = 0;
        for(int i = 0; i < n; i++){
            if(nums[i] == 0){
                res = Math.max(res, cnt);
                cnt = 0;            
            }else{
                cnt++;
            }
        }
        return Math.max(res, cnt);
    }
}